src/extensions/epigenomics/Encode.ttl

```diff
@@ -16,211 +17,133 @@
+  owl:imports <https://datamodel.terra.bio/TerraCore> ;
+  owl:imports <https://datamodel.terra.bio/TerraCoreValueSets> ;
+  owl:imports <https://datamodel.terra.bio/TerraDCAT_ap> ;
   owl:versionInfo "Created with TopBraid Composer" ;
 .
+Encode:ATACseqReplicated
+  a owl:Class ;
+  rdfs:label "ATAC-seq Replicated" ;
+  rdfs:subClassOf Encode:Pipeline ;
+  TerraCore:hasCrossReference "http://encodeproject.org/pipelines/ENCPL787FUN" ;
+.
+Encode:ATACseqUnreplicated
+  a owl:Class ;
+  rdfs:label "ATAC-seq Unreplicated" ;
+  rdfs:subClassOf Encode:Pipeline ;
+  TerraCore:hasCrossReference "http://encodeproject.org/pipelines/ENCPL344QWT" ;
+.
 Encode:AuditFinding
   a owl:Class ;
   rdfs:label "AuditFinding" ;
   rdfs:subClassOf owl:Thing ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:allValuesFrom xsd:integer ;
       owl:onProperty prov:value ;
     ] ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:cardinality "1"^^xsd:nonNegativeInteger ;
       owl:onProperty Encode:hasLevelName ;
     ] ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:minCardinality "1"^^xsd:nonNegativeInteger ;
-      owl:onProperty Encode:wasFoundIn ;
+      owl:onProperty skos:closeMatch ;
     ] ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:minCardinality "1"^^xsd:nonNegativeInteger ;
-      owl:onProperty skos:closeMatch ;
+      owl:onProperty prov:wasGeneratedBy ;
     ] ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:minCardinality "1"^^xsd:nonNegativeInteger ;
-      owl:onProperty prov:wasGeneratedBy ;
+      owl:onProperty Encode:isFoundIn ;
     ] ;
 .
-Encode:ChIP-seq_assay
-  a obo:OBI_0000716 ;
-  rdfs:label "ChIP-seq assay" ;
-.
 Encode:ChIPSeq
   a owl:Class ;
   rdfs:label "ChIPSeq" ;
   rdfs:subClassOf TerraCore:AssayActivity ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:cardinality "1"^^xsd:nonNegativeInteger ;
       owl:onProperty Encode:hasStrandSpecificity ;
     ] ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:cardinality "1"^^xsd:nonNegativeInteger ;
       owl:onProperty TerraCore:usedSample ;
     ] ;
-  owl:equivalentClass [
-      a owl:Restriction ;
-      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
-      owl:onProperty TerraCore:hasAntibody ;
-    ] ;
-  owl:equivalentClass [
+  rdfs:subClassOf [
       a owl:Restriction ;
       owl:minCardinality "1"^^xsd:nonNegativeInteger ;
-      owl:onProperty prov:generated ;
+      owl:onProperty TerraCore:usesAntibody ;
     ] ;
 .
 Encode:ChIPSeqReadMapping
   a owl:Class ;
-  TerraCore:hasCrossReference "http://encodeproject.org/pipelines/ENCPL220NBH" ;
   rdfs:label "ChIP-seq read mapping" ;
   rdfs:subClassOf Encode:Pipeline ;
+  TerraCore:hasCrossReference "http://encodeproject.org/pipelines/ENCPL220NBH" ;
```

Comment:
I was just curious about these changes -
Is this reordering to address what TBC did to the properties?
"TBC did some random reordering of properties"
Should the "rdfs" prefixed properties come before "TerraCore" one?

Reply:
Some of it was due to property names but I can't explain some others.  It's possible that a previous version of TBC or possibly Protege order things differently on export.  But there is definitely some reordering that I cannot explain.